content/en/docs/userguide/try_waypoint.md

```diff
@@ -0,0 +1,136 @@
+
+  Using Kmesh manage default namespace
+
+  ```
+  [root@ ~]# kubectl label namespace default istio.io/dataplane-mode=Kmesh
+  [root@ ~]# kubectl get namespace -L istio.io/dataplane-mode
+  NAME                 STATUS   AGE   DATAPLANE-MODE
+  default              Active   13d   Kmesh
+  istio-system         Active   13d   
+  kmesh-system         Active   27h   
+  kube-node-lease      Active   13d   
+  kube-public          Active   13d   
+  kube-system          Active   13d   
+  local-path-storage   Active   13d   
+  ```
+ 
+Deploy bookinfo:
+
+  ```
+  [root@ ~]# kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.21/samplesbookinfo/platform/kube/bookinfo.yaml
+  ```
+
+Deploy sleep as curl client:
+
+  ```
+  [root@ ~]# kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.21/samples/sleepsleep.yaml
+  [root@ ~]# kubectl get pods
+  NAME                             READY   STATUS    RESTARTS   AGE
+  details-v1-5f4d584748-bz42z      1/1     Running   0          72s
+  productpage-v1-564d4686f-2rjqc   1/1     Running   0          72s
+  ratings-v1-686ccfb5d8-dnzkf      1/1     Running   0          72s
+  reviews-v1-86896b7648-fqm4z      1/1     Running   0          72s
+  reviews-v2-b7dcd98fb-nn42q       1/1     Running   0          72s
+  reviews-v3-5c5cc7b6d-q4r5h       1/1     Running   0          72s
+  sleep-9454cc476-86vgb            1/1     Running   0          62s
+  ```
+
+Test boofinfo works as expected
+
+  ```
+  [root@ ~]# kubectl exec deploy/sleep -- curl -s http://productpage:9080/ | grep -o "<title>.*</title>"
+    <title>Simple Bookstore App</title>
+  ```
+
+Deploy waypoint:
+
+  Deploy a waypoint for service account `bookinfo-reviews`, so any traffic to service `reviews` will be mediated by that waypoint proxy
+
+  ```
+  [root@ ~]# istioctl x waypoint apply --service-account bookinfo-reviews
+  [root@ ~]# kubectl get pods
+  NAME                                               READY   STATUS    RESTARTS   AGE
+  bookinfo-reviews-istio-waypoint-5d544b6d54-v5tc9   1/1     Running   0          4s
+  details-v1-5f4d584748-bz42z                        1/1     Running   0          4m35s
+  productpage-v1-564d4686f-2rjqc                     1/1     Running   0          4m35s
+  ratings-v1-686ccfb5d8-dnzkf                        1/1     Running   0          4m35s
+  reviews-v1-86896b7648-fqm4z                        1/1     Running   0          4m35s
+  reviews-v2-b7dcd98fb-nn42q                         1/1     Running   0          4m35s
+  reviews-v3-5c5cc7b6d-q4r5h                         1/1     Running   0          4m35s
+  sleep-9454cc476-86vgb                              1/1     Running   0          4m25s
+  ```
+  
+  Replace the waypoint image with the Kmesh customized image. Based on istio-proxy, Kmesh adds an customized listener filter called [kmesh_tlv](https://github.com/kmesh-net/waypoint/tree/master/source/extensions/filters/listener/kmesh_tlv) to connect L4 and L7.
```

Comment:
connect L4 and L7 is too wide. It is to parse our private protocol which takes additional target address or metadata, etc.  We should better word it